Cannot find module 'next-international/middleware'

Super happy to see it got /app support.

Uppon checking the example turns out i cannot import the middleware and other objects.

```bash
// [lang]/layout
Module '"next-international"' has no exported member 'I18nProviderServer'.

//middleware
Cannot find module 'next-international/middleware' or its corresponding type declarations.
```

I can surpress the issue by importing from `/dist/middleware` but then i get another error about it not being a named export there.

Package Version: 0.5 from the latest release tag.

--- a/packages/next-international/package.json
+++ b/packages/next-international/package.json
@@ -30,7 +30,10 @@
     "internationalization"
   ],
   "files": [
-    "dist"
+    "dist",
+    "client.d.ts",
+    "server.d.ts",
+    "middleware.d.ts"
   ],
   "repository": {
     "type": "git",

